cmd/helios-cli/cli_smoke_test.go

```diff
@@ -72,3 +72,17 @@ func TestCLI_Help(t *testing.T) {
 	}
 	_ = os.Stdout
 }
+
+func TestCLI_Version(t *testing.T) {
+    bin := filepath.Join(t.TempDir(), "helios-cli.testbin")
+    if out, err := exec.Command("go", "build", "-o", bin, ".").CombinedOutput(); err != nil {
```

Comment:
Suggestion: consider adding `-ldflags` in this build to validate that version/commit/date variables are wired end-to-end.